lib/cmps/sourcepoint-frame.ts

```diff
@@ -36,6 +36,18 @@ export default class SourcePoint extends AutoConsentCMPBase {
   }
 
   async detectPopup() {
+
+    // we only return true if the manager is open and we can actually perform an opt-out
```

Comment:
Will it will break the opt-in flow? Is it possible to make an "actionable" condition that would work for both?

Reply:
Good question. It seems that optOut is waiting for `.sp_choice_type_12`, whereas optIn is waiting for `.sp_choice_type_11`. So if we would accept either one of them in the `detectPopup()` function we would still risk that the opt-out doesn't work.

Having an `actionable()` function sounds like a great idea, I just didn't want to make a large change now in the flow. I think in any case we need more information in the `popupFound` message to distinguish these two cases, as we only want to ask the user whether to turn this on if the opt-out is actionable.

Reply:
1. I think it makes sense to study the existing examples of these popups (we have quite a few of them in the test suite), and see if we can compose another selector

2. Note that if there's no reject button, we may be able to just hide it and still return true.

3. Adding more fields in the messages makes sense, but it would also require changes all over the place, so it needs to be thought through

Reply:
If I understood correctly, the actual problem on the problematic website is related not to sourcepoint popup, but with a separate paywall popup, isn't it?
I understand that this fix will mitigate the side effect, but it's not clear if this ever happens on other websites. If it's meant as a site-specific mitigation, would it be better to use the privacy config?